docs/reference/search/uri-request.asciidoc

```diff
@@ -21,19 +21,19 @@ And here is a sample response:
     "timed_out": false,
     "took": 62,
     "_shards":{
-        "total" : 5,
-        "successful" : 5,
+        "total" : 1,
```

Comment:
These changes came from me shrinking the twitter example to a single shard. It made the term suggester math work out much better and I don't think it hurts the docs in general.